src/data/maps/indoors_b.asm

```diff
@@ -255,4 +255,4 @@ dw IndoorsBFB  ; $2FCE9
 dw IndoorsBFC  ; $2FD1C
 dw IndoorsBFD  ; $2FD88
 dw IndoorsBFE  ; $2FDCA
-dw $7E01  ; Invalid room pointer
+dw $7DD7  ; Invalid room pointer
```

Comment:
Probably still points to something, considering it changed between versions

Reply:
I think it just points after `IndoorsBFE`, but there is not more data there for an actual room.